api/Makefile

```diff
@@ -4,10 +4,10 @@
 include ../Makefile-modules.mk
 
 test:
-	go test -v -timeout 45m -cover ./... -ldflags "-X sigs.k8s.io/kustomize/api/provenance.version=v444.333.222"
+	go test -v -timeout 45m -cover ./...
```

Comment:
I removed this because the test `TestAddManagedbyLabel` was not being impacted by it as far as I could tell. When I checked on master it failed as well.